### PR TITLE
fixes #806

### LINKED ISSUE #806: Inaccuracy in Spring Boot health check example
In section [2.4.4. Interacting with the Unmodified Spring Boot Tomcat Booster](https://doc-stage.usersys.redhat.com/documentation/en-us/red_hat_openshift_application_runtimes/1/html-single/spring_boot_tomcat_runtime_guide/#interacting_with_the_unmodified_spring_boot_tomcat_booster_3), step 2 tells user to call `/api/stop`, follow it by `/api/greeting` and expect the following outcome:
```
<html>
  <head><title>Error</title></head>
  <body>503 - Service Unavailable</body>
</html>
```

It is not accurate as following stop request by greeting immediately will most likely result in `404` error as OpenShift might not have noticed the issue yet. Once the OpenShift takes the pod out, the expected outcome is also different from the one specified. The actual outcome is an http page like this:
```
<html>
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1">

  <style type="text/css">
  /*!
   * Bootstrap v3.3.5 (http://getbootstrap.com)
   * Copyright 2011-2015 Twitter, Inc.
   * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
   */
  /*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
  html {
    font-family: sans-serif;
    -ms-text-size-adjust: 100%;
    -webkit-text-size-adjust: 100%;
  }
  body {
    margin: 0;
  }
  h1 {
    font-size: 1.7em;
    font-weight: 400;
    line-height: 1.3;
    margin: 0.68em 0;
  }
  * {
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box;
  }
  *:before,
  *:after {
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box;
  }
  html {
    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
  }
  body {
    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
    line-height: 1.66666667;
    font-size: 13px;
    color: #333333;
    background-color: #ffffff;
    margin: 2em 1em;
  }
  p {
    margin: 0 0 10px;
    font-size: 13px;
  }
  .alert.alert-info {
    padding: 15px;
    margin-bottom: 20px;
    border: 1px solid transparent;
    background-color: #f5f5f5;
    border-color: #8b8d8f;
    color: #363636;
    margin-top: 30px;
  }
  .alert p {
    padding-left: 35px;
  }
  a {
    color: #0088ce;
  }

  ul {
    position: relative;
    padding-left: 51px;
  }
  p.info {
    position: relative;
    font-size: 15px;
    margin-bottom: 10px;
  }
  p.info:before, p.info:after {
    content: "";
    position: absolute;
    top: 9%;
    left: 0;
  }
  p.info:before {
    content: "i";
    left: 3px;
    width: 20px;
    height: 20px;
    font-family: serif;
    font-size: 15px;
    font-weight: bold;
    line-height: 21px;
    text-align: center;
    color: #fff;
    background: #4d5258;
    border-radius: 16px;
  }

  @media (min-width: 768px) {
    body {
      margin: 4em 3em;
    }
    h1 {
      font-size: 2.15em;}
  }

  </style>
  </head>
  <body>
    <div>
      <h1>Application is not available</h1>
      <p>The application is currently not serving requests at this endpoint. It may not have been started or is still starting.</p>

      <div class="alert alert-info">
        <p class="info">
          Possible reasons you are seeing this page:
        </p>
        <ul>
          <li>
            <strong>The host doesn't exist.</strong>
            Make sure the hostname was typed correctly and that a route matching this hostname exists.
          </li>
          <li>
            <strong>The host exists, but doesn't have a matching path.</strong>
            Check if the URL path was typed correctly and that the route was created using the desired path.
          </li>
          <li>
            <strong>Route and path matches, but all pods are down.</strong>
            Make sure that the resources exposed by this route (pods, services, deployment configs, etc) have at least one pod running.
          </li>
        </ul>
      </div>
    </div>
  </body>
</html>
```

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

**Comment by rhoads-zach** [comment]
@gytis mind taking a look?

File: docs/topics/health-check-mission-basic-interaction-spring-boot-tomcat.adoc [hunk 1]
```diff
@@ -33,10 +33,21 @@ $ curl http://{app-name}-{project-name}.{os-route-hostname}/api/stop
...
+      <h1>Application is not available</h1>
+      ...
+    </div>
+  </body>
 </html>
 ----
++
+NOTE: Depending on when OpenShift removes the pod after you invoke the /api/stop endpoint, you may initially see a 404 error code. If continue to invoke the /api/greeting endpoint, you will see the `Application is not available` page after OpenShift removes the pod.
+
+It is not accurate as following stop request by greeting immediately will most likely result in 404 error as OpenShift might not have noticed the issue yet. Once the OpenShift takes the pod out, the expected outcome is also different from the one specified.
```
Comment (gytis) [inline docs/topics/health-check-mission-basic-interaction-spring-boot-tomcat.adoc]: I don't think that this was meant to be here :)

Comment (rhoads-zach) [reply docs/topics/health-check-mission-basic-interaction-spring-boot-tomcat.adoc]: Nice, catch. sorry, copy/paste error.

**Comment by rhoads-zach** [comment]
@gytis fixed.

File: docs/topics/health-check-mission-basic-interaction-spring-boot-tomcat.adoc [hunk 2]
```diff
@@ -33,10 +33,19 @@ $ curl http://{app-name}-{project-name}.{os-route-hostname}/api/stop
...
+  <body>
+    <div>
+      <h1>Application is not available</h1>
+      ...
+    </div>
+  </body>
 </html>
 ----
++
+NOTE: Depending on when OpenShift removes the pod after you invoke the /api/stop endpoint, you may initially see a 404 error code. If continue to invoke the /api/greeting endpoint, you will see the `Application is not available` page after OpenShift removes the pod.
```
Comment (tradej) [inline docs/topics/health-check-mission-basic-interaction-spring-boot-tomcat.adoc]: _/api/stop_ and _/api/greeting_ should be verbatim, i. e. surrounded by backticks.

Comment (rhoads-zach) [reply docs/topics/health-check-mission-basic-interaction-spring-boot-tomcat.adoc]: was keeping consistency with the topic, but agreed. Fixing in the whole file.

**Comment by tradej** [comment]
retest this please

**Comment by tradej** [comment]
OK, merging...

**Comment by tradej** [comment]
merged 43ec22e3a

**Pull request closed without merging**